Monika After Story/game/script-farewells.rpy

```diff
@@ -44,3 +44,62 @@ label bye_untilwemeetagain:
     m 2e "'{i}Goodbyes are not forever, Goodbyes are not the end. They simply mean I’ll miss you, Until we meet again.{/i}'"
     m "Ehehe, 'till then, [player]!"
     return 'quit'
+    
+init 5 python:
+    addEvent(Event(persistent.farewell_database,eventlabel="i'll_see_you",random=True),eventdb=evhand.farewell_database)
+    
+label i'll_see_you:
```

Comment:
Apostrophes cannot be part of labels. Also please prefix all farewell labels with `bye_`